src/services/favoritesService.js

```diff
@@ -0,0 +1,25 @@
+// import { StatusCodes } from 'http-status-codes';
+import favoriteModel from '../models/favorite';
+
+export const addToFavorite = async (data) => {
+  const { userId, houseId } = data;
+
```

Comment:
You could check if userId and houseId exist.
```suggestion

if (!userId || !houseId) {
  return;
}
```